### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by opsline-josiecki**

### CODE REVIEW

File: lessServer.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ var port = 8000;
  */
 http.createServer(function (req, res) {
     handleCompileLessFileRequest (req, res);
-}).listen(port, ip);
+}).listen(port);//, ip);
```
Comment (gdakram) [inline lessServer.js]: Please remove commented out code.

File: lessServer.js [hunk 2]
```diff
@@ -62,11 +62,11 @@ function compileLessFile(res, lessFileContents, lessFilePath, cssFilePath, paths
                      if (compilationError) {
                          // Log Paths to help with debuggin - often an issue with include precedence
                          console.log("PATHS (in order of precedence)): \n" + paths.split(',').join("\n"));
-                         less.writeError(compilationError);
+                         console.error(compilationError);
                          return reqErr(res, compilationError, "LESS PARSER ERROR: Line " + compilationError.line + ', ' + compilationError.message + '. File: ' + lessFilePath);
                      }
                      if (compileToFile) {
-                         writeCompiledOutputToCssFile(res, compiledCssOutput, cssFilePath);
+                         writeCompiledOutputToCssFile(res, compiledCssOutput.css, cssFilePath);
```
Comment (gdakram) [inline lessServer.js:69]: The `.css`, is that a typo?

**Review by gdakram — CHANGES_REQUESTED** [review]

File: lessServer.js (hunk 2 above)
Comment (opsline-josiecki) [reply lessServer.js:69]: no, there seems to be a css string in the object https://worldsurfleague.slack.com/archives/GN7PETRTJ/p1585580998031600?thread_ts=1585092046.135400&cid=GN7PETRTJ

File: lessServer.js (hunk 1 above)
Comment (opsline-josiecki) [reply lessServer.js]: you got it

File: lessServer.js (hunk 2 above)
Comment (gdakram) [reply lessServer.js:69]: I see. In that case, can we lock in the version for less?
```
RUN npm install less
```

We might run into other future incompatibilities.

Comment (opsline-josiecki) [reply lessServer.js:69]: 👍

**Pull request merged**